pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -546,6 +558,22 @@ func checkAndPublishDhcpLeases(ctx *zedrouterContext) {
 	}
 }
 
+func isEmptyIP(ip net.IP) bool {
+	return ip.Equal(net.IP{})
+}
+
+func ipListChanged(one []net.IP, two []string) bool {
```

Comment:
Other functions are called Equal and returns the inverse value. That would be more familiar to a reader I think.

Reply:
changed the function name to ipListEqual that returns the inverse value.